src/routes/admin/screenmessage/+page.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  import { pb } from "$lib/stores/authStore";
+
+  export let data;
+  let screenMessages = data.screenMessages;
+  let activeMessage = data.activeMessage[0];
+
+  let open = !activeMessage.isVisible;
+  async function updateActiveMessage() {
+    await pb.collection("activeMessage").update(activeMessage.id, {
+      message: selectedMessage.id,
+      isVisible: !open
+    });
+  }
+
+  let selectedMessage;
+
+  async function updateScreenMessages() {
+    updateActiveMessage();
+    for (let message of screenMessages) {
+      await pb.collection("screenMessages").update(message.id, {
+        title: message.title,
+        subtext: message.subtext,
+        isVisible: message.id == selectedMessage.id
```

Comment:
Du nevnte at du brukte en relation for å vite hvilken som er aktiv. Er det da nødvendig å ha denne kolonnen?

Reply:
`isVisible` er kun for å bestemme hvilken radio knapp som skal være checked når man laster siden.

Reply:
Bør ikke dette være mulig å vite utifra hva activeMessage sier, uten å ha en for hver melding?